charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/values.yaml

```diff
@@ -1,6 +1,5 @@
 replicas: 1
 clusterName: shoot-foo-bar
-kubernetesVersion: 1.7.5
```

Comment:
Can we stop passing the `kubernetesVersion` in https://github.com/gardener/gardener-extension-provider-alicloud/blob/281cca7e059a4706d0ad685939702b85bce06d93/pkg/controller/controlplane/valuesprovider.go#L402-L436?